Networking/NetworkingTests/Mapper/ReportOrderTotalsMapperTests.swift

```diff
@@ -0,0 +1,55 @@
+import XCTest
+@testable import Networking
+
+final class ReportOrderTotalsMapperTests: XCTestCase {
+
+    /// Dummy Site ID.
+    ///
+    private let dummySiteID: Int64 = 123
+
+    private let fileNameWithDataEnvelope = "report-orders-total"
+    private let fileNameWithoutDataEnvelope = "report-orders-total-without-data"
+
+    func test_order_statuses_is_mapped_from_response_with_data_envelope() {
+        // Given
+        let mapper = ReportOrderTotalsMapper(siteID: dummySiteID)
+        guard let data = Loader.contentsOf(fileNameWithDataEnvelope) else {
+            XCTFail(fileNameWithDataEnvelope + ".json not found")
+            return
+        }
+
+        // When
+        let statuses = try? mapper.map(response: data)
```

Comment:
nit: can probably make the result non-nil by using `try XCTUnwrap`

Reply:
Also updated in 6dad5975a61e0c23e254c79e1fc5d3129a99e851.